Common/src/grid_movement_structure.cpp

```diff
@@ -7755,7 +7815,16 @@ void CSurfaceMovement::WriteFFDInfo(CGeometry *geometry, CConfig *config) {
       output_file << "FFD_DEGREE_I= " << FFDBox[iFFDBox]->GetlOrder()-1 << endl;
       output_file << "FFD_DEGREE_J= " << FFDBox[iFFDBox]->GetmOrder()-1 << endl;
       if (nDim == 3) output_file << "FFD_DEGREE_K= " << FFDBox[iFFDBox]->GetnOrder()-1 << endl;
-      
+      if (config->GetFFD_Blending() == BSPLINE_UNIFORM) {
```

Comment:
You're adding a new option in the FFD info of the mesh for the blending.. it looks like it is backwards compatible, i.e., if you don't put it, the default is the existing behavior, correct? Also, are there any error checks on the reading of the spline order (or initialization of a default for spline order somewhere)?